docs/reference/data-streams/set-up-a-data-stream.asciidoc

```diff
@@ -4,48 +4,74 @@
 
 To set up a data stream, follow these steps:
 
-. <<configure-a-data-stream-ilm-policy>>.
-. <<create-a-data-stream-template>>.
-. <<create-a-data-stream>>.
-. <<secure-a-data-stream>>.
+* <<create-index-lifecycle-policy>>
+* <<create-component-templates>>
+* <<create-index-template>>
+* <<create-data-stream>>
+* <<secure-data-stream>>
 
-You can also <<convert-an-index-alias-to-a-data-stream,convert an index alias to
+You can also <<convert-index-alias-to-data-stream,convert an index alias to
 a data stream>>.
 
 [discrete]
-[[configure-a-data-stream-ilm-policy]]
-=== Optional: Configure an {ilm-init} lifecycle policy
+[[create-index-lifecycle-policy]]
+=== Step 1. Create an index lifecycle policy
 
-While optional, we recommend you configure an <<set-up-lifecycle-policy,{ilm}
-({ilm-init}) policy>> to automate the management of your data stream's backing
-indices.
+{ilm-init} uses an index lifecycle policy to automate the management of your
+data stream's backing indices.
 
-In {kib}, open the menu and go to *Stack Management > Index Lifecycle Policies*.
-Click *Create policy*.
+To create an index lifecycle policy in {kib}, open the menu and go to *Stack
+Management > Index Lifecycle Policies*. Click *Create policy*.
 
-[role="screenshot"]
-image::images/ilm/create-policy.png[Create Policy page]
-
-[%collapsible]
-.API example
-====
-Use the <<ilm-put-lifecycle,create lifecycle policy API>> to configure a policy:
+You can also use the <<ilm-put-lifecycle,create lifecycle policy API>>.
 
 [source,console]
 ----
-PUT /_ilm/policy/my-data-stream-policy
+PUT _ilm/policy/my-lifecycle-policy
 {
   "policy": {
     "phases": {
       "hot": {
         "actions": {
           "rollover": {
-            "max_primary_shard_size": "25GB"
+            "max_age": "30d",
+            "max_size": "50gb",
+            "max_primary_shard_size": "25gb"
+          },
+          "set_priority": {
+            "priority": 100
           }
         }
       },
-      "delete": {
+      "warm": {
         "min_age": "30d",
+        "actions": {
+          "shrink": {
+            "number_of_shards": 1
+          },
+          "forcemerge": {
+            "max_num_segments": 1
+          }
```

Comment:
It may make sense to move one or both of these to the hot phase.

I'm also not entirely sure if the order provided is correct. From what I understand, a shrink actually recreates the index so maybe a force merge isn't necessary? If that's the case, I can move force merge up to `hot`.